Emma/emma_libs/memoryManager.py

```diff
@@ -208,87 +198,116 @@ def createStandardReports():
                             yAxe = yAxe + distanceBetweenElements
-                            plottedElements.append((element[Element.AddressEnd], yAxe))
+                            plottedElements.append((element[Element.addressEnd], yAxe))
                             if yAxe > lastSectionPosition:
                                 lastSectionPosition = yAxe
-                            if element[Element.AddressEnd] > biggestEndAddress:
-                                biggestEndAddress = element[Element.AddressEnd]
-                        elif element[Element.AddressStart] < biggestEndAddress:
-                            for el in plottedElements:
-                                if element[Element.AddressStart] < el[PlottedElement.AddressEnd] and yAxe <= el[PlottedElement.yAxe]:
-                                    yAxe = el[PlottedElement.yAxe] + distanceBetweenElements
+                            if element[Element.addressEnd] > biggestEndAddress:
+                                biggestEndAddress = element[Element.addressEnd]
+                        elif element[Element.addressStart] < biggestEndAddress:
+                            for plottedElement in plottedElements:
+                                if element[Element.addressStart] < plottedElement[PlottedElement.addressEnd] and yAxe <= plottedElement[PlottedElement.yAxe]:
+                                    yAxe = plottedElement[PlottedElement.yAxe] + distanceBetweenElements
                         else:
                             yAxe = y
-                            biggestEndAddress = element[Element.AddressEnd]
+                            biggestEndAddress = element[Element.addressEnd]
                     image.add(image.rect((xAxeRectStart, yAxe), size=(rectLength, 10), stroke=colour, fill=colour, opacity='0.3'))
-                    # check if the FQN fits in the rectangle
-                    if rectLength <= len(element[Element.Fqn]):
-                        image.add(image.text(element[Element.Fqn], insert=(xAxeRectStart, yAxe - 1), font_size='2px', writing_mode="lr", font_family="Helvetica, sans-serif", fill=fontColour))
+                    # Check if the FQN fits in the rectangle
+                    if rectLength <= len(element[Element.fqn]):
+                        image.add(image.text(element[Element.fqn], insert=(xAxeRectStart, yAxe - 1), font_size='2px', writing_mode="lr", font_family="Helvetica, sans-serif", fill=fontColour))
                         xAxeStart = xAxeRectStart + 1
-                        xAxeEnd = element[Element.AddressEnd] - startPoint - 1
-                        # if the rectangle smaller than 4, then write the end address outside the rectangle
+                        xAxeEnd = element[Element.addressEnd] - startPoint - 1
+                        # If the rectangle smaller than 4, then write the end address outside the rectangle
```

Comment:
```suggestion
                        # If the rectangle smaller than 4px, then write the end address outside the rectangle
```

Make it more dynamic: i.e. 2x font size (https://github.com/bmwcarit/Emma/blob/87c3402c35dfb19499797db8c889f03fbbd6aa1e/Emma/emma_libs/memoryManager.py#L263)

1. Define a font size variable at the top of the function
2. `font_size=str(fontSize)+"px"`